testsuite/MDAnalysisTests/analysis/test_psa.py

```diff
@@ -50,11 +53,15 @@ def _run(self):
         self.frech_dists = self.frech_matrix[self.iu1]
 
     def tearDown(self):
+        try:
```

Comment:
You should not need to delete the oudir explicitly. tempdir will autodestruct as soon as the last reference is gone so your
```python
del self.tempfile
```
should be enough.